src/scrabbleScore.js

```diff
@@ -24,14 +24,23 @@ let SCRABBLE_TILE_SCORES = {
  */
 
 function scrabbleScore(word) {
-  // This is your job. :)
+
+let scrabble_Score=0;
```

Comment:
Whatever naming convention you use, be consistent. JS convention prefers `camelCase` (like Java, C++, etc.). The alternative is `snake_case`.

Use one or the other; you'd never see `scrabble_Score`.